jsonnet/main.jsonnet

```diff
@@ -6,7 +6,7 @@ local addBearerTokenToServiceMonitors = (import './utils/add-bearer-token-to-ser
 
 local alertmanager = import './components/alertmanager.libsonnet';
 local alertmanagerUserWorkload = import './components/alertmanager-user-workload.libsonnet';
-local grafana = import './components/grafana.libsonnet';
+local dashboard = import './components/dashboard.libsonnet';
```

Comment:
(nit) I would use `dashboards` rather than `dashboard` everywhere as it reads better for me.